scripts/create-tree.ts

```diff
@@ -44,6 +44,11 @@ async function main() {
       validInputFile.windowIndex
     );
 
+  // Remove metadata object to reduce output file size
+  Object.values(recipientsWithProofs).forEach(
+    (recipientData) => (recipientData.metadata = {})
+  );
+
```

Comment:
can this removal be done in the scraper instead of here?

Reply:
Done here: https://github.com/across-protocol/scraper-api/pull/493